src/classes/traveler.js

```diff
@@ -0,0 +1,46 @@
+import User from './user';
+
+class Traveler extends User {
+  constructor(travelers, trips, destinations, traveler, today) {
+    super(travelers, trips, destinations, today)
+    this.id = traveler.id;
+    this.name = traveler.name;
+    this.travelerType = traveler.travelerType;
+  };
+
+  getMyTrips() {
+    return this.destinations.reduce((trips, destination) => {
+      this.trips.find(trip => (this.id === trip.userID && destination.id === trip.destinationID) && trips.push(destination));
```

Comment:
I would break this out into 2 lines instead of using the &&, just for readability.